src/common/filter/test/ExpressionTest.cpp

```diff
@@ -735,4 +735,42 @@ TEST_F(ExpressionTest, InvalidExpressionTest) {
 #undef TEST_EXPR
 }
 
+
+TEST_F(ExpressionTest, StringLengthLimitTest) {
```

Comment:
Perhaps this testcase should be better placed in parser/test/ParserTest.cpp?

Reply:
Emmm, I have been trying to figure out a way to test `Expression` individually, instead of testing from inside a complete statement. But it seems no easy way to do this with Bison, unless we allow to accept an ordinary expression as a whole statement in the gramar. Any idea?